src/coverlet.core/CoverageResult.cs

```diff
@@ -41,6 +41,7 @@ public class CoverageResult
     {
         public string Identifier;
         public Modules Modules;
+        public bool IsSourceLinkUsed;
```

Comment:
nit: I prefer to maintain switch name `UseSourceLink`

Reply:
Done